<a href="https://colab.research.google.com/github/alamirrangel/curso_ebac_frontend/blob/exercicio1_python/exercicio1_mod1_python_ebac.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

1) Exercício Olá, mundo.

In [ ]:
print("Olá, mundo")

Olá, mundo


2) Exercício Números

Calculo de ticket medio

Valor Total Vendas = svv

Qtd Total Vendas = sqv

Ticket Medio = tkt

tkt=svv/sqv

Planilha:

| Dia   | Valor Total Vendas | Qtd Total Vendas | Ticket Medio |
|-------|--------------------|------------------|-|
| 19/01 | 153.98             | 3                |?|
| 20/01 | 337.01             | 7                |?|
| 23/01 | 295.33             | 5                |?|

In [ ]:
svv_19 = 153.98
sqt_19 = 3
tkt_19 = svv_19 / sqt_19
print(tkt_19)

svv_20 = 337.01
sqt_20 = 7
tkt_20 = svv_20 / sqt_20
print(tkt_20)

svv_23 = 295.33
sqt_23 = 5
tkt_23 = svv_23 / sqt_23
print(tkt_23)

51.32666666666666
48.144285714285715
59.065999999999995


| Dia   | Valor Total Vendas | Qtd Total Vendas | Ticket Medio |
|-------|--------------------|------------------|-|
| 19/01 | 153.98             | 3                |51.32666666666666|
| 20/01 | 337.01             | 7                |48.144285714285715|
| 23/01 | 295.33             | 5                |59.065999999999995|

3) Exercício String

